domain/src/main/java/org/oppia/android/domain/question/QuestionConstantsProvider.kt

```diff
@@ -21,4 +30,16 @@ class QuestionModule {
   @Provides
   @QuestionTrainingSeed
   fun provideQuestionTrainingSeed(oppiaClock: OppiaClock): Long = oppiaClock.getCurrentTimeMs()
+
+  @Provides
+  @ViewHintPenalty
+  fun provideViewHintPenalty(): Double = 0.1
+
+  @Provides
+  @WrongAnswerPenalty
+  fun provideWrongAnswerPenalty(): Double = 0.1
```

Comment:
For this & above: it feels a bit like the unit is missing here. Maybe 'WrongAnswerScorePenalty' would be clearer?

Reply:
Fixed